Packages/MIES/MIES_NeuroDataWithoutBorders.ipf

```diff
@@ -33,6 +33,8 @@
 ///     + NWB_AppendSweepLowLevel()
 ///     + NWB_Flush()
 
+static Constant NWB_ASYNC_WRITING_TIMEOUT = 5
```

Comment:
@t-b was this timeout tested with the max sweep length allowed across max headstages?

Reply:
@timjarsky No, but your approach would also force us to use a timeout like 15 minutes or so. You can use 32 devices with 8 headstages and 8 TTL channels where we acquire 30 minute sweeps everywhere.

Reply:
The fallout from a too short timeout is that the user gets an annoying assertion message while the preemptive thread continues to write out data.

Reply:
@timjarsky I thought about it again. And I reworked it, thanks for the comment.